scipy/interpolate/tests/test_bsplines.py

```diff
@@ -2170,8 +2178,10 @@ def test_design_matrix(self):
         t3, c3, k = self.make_3d_case()
 
         xi = np.asarray([[1, 2, 3], [4, 5, 6]])
-        dm = NdBSpline(t3, c3, k)(xi)
+        dm = NdBSpline(t3, c3, k).design_matrix(xi, t3, k)
+        dm1 = NdBSpline.design_matrix(xi, t3, [k, k, k])
         assert dm.shape[0] == xi.shape[0]
+        assert_allclose(dm.todense(), dm1.todense(), atol=1e-16)
```

Comment:
What do these three lines change, since I'm not an expert on it? Is it just covering some line(s) that I noticed weren't covered or something?

Reply:
It's just that your review made me see that `test_design_matrix` method should probably actually call `.design_matrix` :-). The design matrix itself is indirectly tested through RGI, so we can remove the test if wanted. The test currently just checks that 1) the number of rows in the design matrix equals the number of data points --- this was the original intent, --- and 2) that the `k` argument can be either a scalar or a three-tuple, see the other comment.